tools/test-framework/src/bootstrap/binary/chain.rs

```diff
@@ -160,7 +160,7 @@ pub fn bootstrap_foreign_client<ChainA: ChainHandle, ChainB: ChainHandle>(
     let foreign_client =
         ForeignClient::restore(ClientId::default(), chain_b.clone(), chain_a.clone());
 
-    let event = foreign_client.build_create_client_and_send(&Default::default())?;
```

Comment:
Can we add a new test? Would like to assert that:

1. default settings specified in the config.toml for source chain is respected when we create a new client

2. we correctly override the default settings with custom ones.